scripts/zones/Balgas_Dais/mobs/Bitoso.lua

```diff
@@ -11,12 +11,12 @@ local entity = {}
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.HP_HEAL_CHANCE, 90)
     mob:setMobMod(xi.mobMod.HEAL_CHANCE, 100)
-    mob:setMod(xi.mod.REGEN, 0)
 end
 
 entity.onMobSpawn = function(mob)
     mob:setMobMod(xi.mobMod.MAGIC_COOL, 20)
     mob:setMobMod(xi.mobMod.SOUND_RANGE, 13)
+    mob:setMod(xi.mod.REGEN, 0)
```

Comment:
what's giving him regen that requires us to set this to 0 onspawn? I believe all mods are by default reset when they die.

Reply:
He has 10/tick regen (from where, I don't know) but also the FFXIclopedia entry explicitly says he should have 0 passive regen, so I didn't look into it further

Reply:
Interesting! I couldn't really find anything either. I'll make not of this and try and find where it's coming from later.